docs/package.json

```
@@ -22,24 +22,23 @@
     "@babel/plugin-transform-object-assign": "^7.24.1",
     "@babel/runtime": "^7.24.4",
     "@babel/runtime-corejs2": "^7.24.4",
+    "@base_ui/react": "workspace:*",
     "@docsearch/react": "^3.6.0",
     "@emotion/cache": "^11.11.0",
     "@emotion/react": "^11.11.4",
     "@emotion/server": "^11.11.0",
     "@emotion/styled": "^11.11.5",
-    "@base_ui/react": "workspace:*",
-    "@mui/base": "5.0.0-beta.40",
-    "@mui/docs": "https://pkg.csb.dev/mui/material-ui/commit/19d7243c/@mui/docs",
-    "@mui/icons-material": "5.15.15",
+    "@mui/docs": "6.0.0-alpha.4",
+    "@mui/icons-material": "6.0.0-alpha.4",
     "@mui/internal-markdown": "^1.0.3",
     "@mui/internal-scripts": "^1.0.6",
-    "@mui/joy": "5.0.0-beta.32",
-    "@mui/material": "5.15.15",
-    "@mui/material-nextjs": "5.15.11",
-    "@mui/styles": "5.15.15",
-    "@mui/system": "5.15.15",
-    "@mui/types": "7.2.14",
-    "@mui/utils": "5.15.14",
+    "@mui/joy": "5.0.0-beta.37",
+    "@mui/material": "6.0.0-alpha.4",
+    "@mui/material-nextjs": "6.0.0-alpha.0",
+    "@mui/styles": "6.0.0-alpha.4",
+    "@mui/system": "6.0.0-alpha.4",
+    "@mui/types": "7.0.0-rc.1",
```

This doesn't look right. 7.2.14 is the latest version in https://github.com/mui/material-ui

```suggestion
    "@mui/types": "7.2.14",
```